packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2281,12 +2315,27 @@ mixin WidgetInspectorService {
   }
 
   /// All events dispatched by a [WidgetInspectorService] use this method
-  /// instead of calling [developer.postEvent] directly so that tests for
-  /// [WidgetInspectorService] can track which events were dispatched by
-  /// overriding this method.
+  /// instead of calling [developer.postEvent] directly.
+  ///
+  /// This allows tests for [WidgetInspectorService] to track which events were
+  /// dispatched by overriding this method.
+  @protected
+  void postEvent(
+    String eventKind,
+    Map<Object, Object?> eventData,
+    {String stream = 'Extension',}
```

Comment:
sadly this isn't formatted correctly. I beleive can trigger running the formatter on just some selected code.